rcl_logging_spdlog/CMakeLists.txt

```diff
@@ -45,12 +45,15 @@ install(TARGETS ${PROJECT_NAME} EXPORT ${PROJECT_NAME}
 
 if(BUILD_TESTING)
   find_package(ament_lint_auto REQUIRED)
+  # cppcheck 1.90 doesn't understand some of the syntax in spdlog's bundled fmt
+  # code.  Since we already have cppcheck disabled on Linux, just disable it
```

Comment:
nit extra space

Reply:
Heh, that's a matter of taste (and age).  I was always taught in school to put two spaces after a sentence, though I understand this is out of vogue nowadays.

I'm not going to change it (just so I don't have to rerun CI).